documentation/src/docs/asciidoc/user-guide/running-tests.adoc

```diff
@@ -156,43 +148,46 @@ for a comprehensive list of options.
 [[running-tests-build-gradle-bom]]
 ===== Aligning dependency versions
 
+TIP: See <<running-tests-build-spring-boot>> for details on how to override the version
+of JUnit used in your Spring Boot application.
+
 Unless you're using Spring Boot which defines its own way of managing dependencies, it is
 recommended to use the JUnit Platform BOM to align the versions of all JUnit 5 artifacts.
 
 [source,groovy,indent=0]
 [subs=attributes+]
+.Explicit platform dependency on the BOM
 ----
 dependencies {
 	testImplementation(platform("org.junit:junit-bom:{bom-version}"))
+    testImplementation("org.junit.jupiter:junit-jupiter")
+    testRuntimeOnly("org.junit.platform:junit-platform-launcher")
 }
 ----
 
 Using the BOM allows you to omit the version when declaring dependencies on all artifacts
 with the `org.junit.platform`, `org.junit.jupiter`, and `org.junit.vintage` group IDs.
 
-TIP: See <<running-tests-build-spring-boot>> for details on how to override the version
-of JUnit used in your Spring Boot application.
-
-[[running-tests-build-gradle-config-params]]
-===== Configuration Parameters
-
-The standard Gradle `test` task currently does not provide a dedicated DSL to set JUnit
-Platform <<running-tests-config-params, configuration parameters>> to influence test
-discovery and execution. However, you can provide configuration parameters within the
-build script via system properties (as shown below) or via the
-`junit-platform.properties` file.
+Since all JUnit artifacts declare a
+https://docs.gradle.org/current/userguide/platforms.html[platform] dependency on the BOM,
```

Comment:
We use the acronym "BOM" in several places, but do we actually define what it stands for anywhere?

If not, we should _define_ it the first time it's used -- for example "BOM (bill of materials)".

I think it would also be good to explicitly state that the `junit-bom` artifact is the BOM we're talking about instead of just calling it "the BOM" or "it".

Reply:
Done in 26829cce41ecd32f16382521fa59495465ce1c67.